ci-operator/config/devfile/integration-tests/devfile-integration-tests-main__v4.9.odo.yaml

```diff
@@ -33,7 +33,7 @@ resources:
       memory: 6Gi
 tests:
 - as: integration-devfile-odo-periodic
-  cron: 0 2 * * *
+  cron: 0 2 * * 0
```

Comment:
Isn't it dangerous when we start to change multiple things to the same Sunday date?